src/StbImage.Generated.Png.cs

```diff
@@ -570,7 +598,8 @@ public static int stbi__parse_png_file(stbi__png z, int scan, int req_comp)
 			var palette = stackalloc byte[1024];
 			byte pal_img_n = 0;
 			byte has_trans = 0;
-			var tc = stackalloc byte[] { 0, 0, 0 };
+			var tc = stackalloc byte[3];
+			tc[0] = 0;
```

Comment:
shouldn't other values be initialized to zero too?

Reply:
Yeah I can do that if you'd like. This is how it is in the version currently used in MonoGame back on this commit: https://github.com/StbSharp/StbImageSharp/blob/55e833e7f0f86b247040dabfbad51ca46f164118/src/StbImage.Generated.Png.cs#L557

It only had the first initialized in the older version same as raw_data one in Tga below, but it should be fine to initialize them as 0's just like the newer code was doing. I can update both if you'd prefer!

Reply:
yes, please update both.

Reply:
Updated.